test/gyppies-test.js

```diff
@@ -7,6 +7,8 @@ const fs = require('fs');
 const path = require('path');
 const rimraf = require('rimraf');
 const ninja = require('ninja.js');
+const debug = require('debug')('gyp.js');
+debug.log = console.log.bind(console);
```

Comment:
BTW: It's a valid `debug` config https://github.com/visionmedia/debug/blob/master/example/stdout.js